qa/workunits/rbd/rbd_mirror_stress.sh

```diff
@@ -325,7 +325,8 @@ write_image()
     local duration=$(($RANDOM % 35 + 15))
 
     timeout ${duration}s rbd --cluster ${cluster} -p ${POOL} bench-write \
-	${image} --io-size 4096 --io-threads 8 --io-total 10G --io-pattern rand || true
+	${image} --io-size 4096 --io-threads 8 --io-total 10G --io-pattern rand \
+        --debug-rbd=20 --debug-journaler=20 2> ${TEMPDIR}/rbd-bench-write.log || true
 }
```

Comment:
Is only the latest write usually interesting so overwriting an older log is ok?

I would make it at least `rbd-bench-write.${cluster}.${image}.log` or `rbd-bench-write.${cluster}.${image}.$(date +%s).log` if it is a good idea to keep all logs (or use `2>>`).